src/components/Thread.vue

```diff
@@ -298,7 +298,7 @@ export default {
 #mail-thread-header-fields {
 	// initial width
 	width: 0;
-	padding-left: 70px;
+	padding-left: 16px;
```

Comment:
I'd say I prefer the indented header, but I don't have a strong opinion. :) @ChristophWurst @nimishavijay @GretaD any preference?

Reply:
Indented also looks better to me